packages/SystemsThinkingWorkbench-Core.package/STWTransferConnection.class/instance/performSimulationStep.st

```diff
@@ -1,12 +1,15 @@
 updating
 performSimulationStep
-	
+
 	| particle |
 	
-	self startNode level: self startNode level - self intake.
-	particle := STWParticle new center: (self position + self segment start); color: self startNode fillingMorph color.
-	self particles add: particle.
-	self addMorph: particle.
-	self lastStepTime ifNil: [
-		self startStepping.
-		self lastStepTime: DateAndTime millisecondClockValue.].
+	((self startNode canHold: self startNode level - self intake)
+			and: (self endNode canHold: self endNode level + self output))
```

Comment:
Könnte man hier eine convenience-Methode einführen, die auf dem Node arbeitet, auf dem es läuft? `CanHoldWith: aNumber` und `CanHoldWithout: aNumber` z.B.

This is more of a request for comments than an actual Change request.